react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -255,29 +241,30 @@ function _mapDispatchToProps(dispatch: Function, ownProps): Object {
  *
  * @param {Object} state - Redux state.
  * @param {Props} ownProps - Properties of component.
- * @returns {{
- *      _audioTrack: Track,
- *      _largeVideo: Object,
- *      _styles: StyleType,
- *      _videoTrack: Track
- *  }}
+ * @returns {Object}
  */
 function _mapStateToProps(state, ownProps) {
     // We need read-only access to the state of features/large-video so that the
     // filmstrip doesn't render the video of the participant who is rendered on
     // the stage i.e. as a large video.
     const largeVideo = state['features/large-video'];
     const tracks = state['features/base/tracks'];
-    const id = ownProps.participant.id;
+    const { participant } = ownProps;
+    const id = participant.id;
     const audioTrack
         = getTrackByMediaTypeAndParticipant(tracks, MEDIA_TYPE.AUDIO, id);
     const videoTrack
         = getTrackByMediaTypeAndParticipant(tracks, MEDIA_TYPE.VIDEO, id);
+    const participantCount = getParticipantCount(state);
+    const renderDominantSpeakerIndicator = participant.dominantSpeaker && participantCount > 2;
+    const _isEveryoneModerator = isEveryoneModerator(state);
+    const renderModeratorIndicator = !_isEveryoneModerator && participant.role === PARTICIPANT_ROLE.MODERATOR;
 
     return {
-        _audioTrack: audioTrack,
-        _isEveryoneModerator: isEveryoneModerator(state),
```

Comment:
I really like these renames. I think they better represent what the component uses the info for.